### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by madhon** [yoshiori/linter-golint#3]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\jdouglas\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:810:9)
Package.hasActivationCommands (C:\Users\jdouglas\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:735:20)
<unknown> (C:\Users\jdouglas\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:189:24)
Package.measure (C:\Users\jdouglas\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:167:15)
```

**Comment by jordandm** [comment]
+1

**Comment by idotpi** [comment]
Use activationCommands instead of activationEvents in your package.json Commands should be grouped by selector as follows:

  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
Report Issue
Package.getActivationCommands - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:808:9
Package.hasActivationCommands - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:733:20
<unknown> - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:187:24
Package.measure - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15
Package.load - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:179:12
PackageManager.loadPackage - /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:372:14
Deprecated selectors
No deprecated selectors

**Comment by yoshiori** [comment]
fix bd059260e711c8f254de0592a94571ae3a6eaef5
